docs/source/concepts/artifact-store.rst

```diff
@@ -0,0 +1,24 @@
+.. _artifact_store_concept:
```

Comment:
Suggest naming this as _section_topic. so in this case, _concepts_artifact_store.